src/App.vue

```diff
@@ -765,6 +765,25 @@ export default class App extends Mixins(AuthApiMixin, CommonMixin, DateMixin, Fi
 
     this.setIsSaving(false)
   }
+
+  /**
+   * Helper to show the confirm dialogs.
+   * @param title The title content in dialog header
+   * @param message The content body
+   * @param action The action label
+   * @param cancel The cancel label
```

Comment:
Clever boy!

I'm going to copy some of what you're done here. Just a couple of changes:
1. rename `action` to `yes`
2. rename `cancel` to `no`
3. use `no` option instead of cancel (which resolves the promise as False instead of rejecting it); subtle difference

Reply:
(*) resolving to False means the result of open() below is false, whereas rejecting means to throw an error, which the catch() handles

Reply:
(**) No is a valid negative action, where Cancel is "not an action".

If the dialog supported pressing Esc to close it, or a top-right "X" button, then that would be cancel.

Reply:
...Meanwhile, in a few cases in Create UI, I'm naming the No button "Cancel" 🙄 But semantically this is correct: the user is selecting a valid negative action instead of dismissing (canceling) the dialog.

Functionally this makes no difference, but it's more correct FWIW.

Reply:
Thanks! 
Sounds good, will update